sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/SonarScannerInstaller.java

```diff
@@ -67,30 +66,30 @@ void doInstall(Version runnerVersion, File toDir) {
     File zipFile = locateZip(runnerVersion);
 
     if (zipFile == null || !zipFile.exists()) {
-      throw new IllegalArgumentException("Unsupported sonar-runner version: " + runnerVersion);
+      throw new IllegalArgumentException("Unsupported sonar-scanner version: " + runnerVersion);
     }
 
     try {
       ZipUtils.unzip(zipFile, toDir);
     } catch (Exception e) {
-      throw new IllegalStateException("Fail to unzip sonar-runner " + runnerVersion + " to " + toDir, e);
+      throw new IllegalStateException("Fail to unzip sonar-scanner " + runnerVersion + " to " + toDir, e);
     }
   }
 
-  private File locateZip(Version runnerVersion) {
+  private File locateZip(Version scannerVersion) {
     File zipFile = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "zipFile". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1854)